inst/sensorreading/sensorreading.Rmd

```diff
@@ -62,8 +62,11 @@ if(!identical(tbl, "The dataset requested is empty.")){
 ***
 
 ```{r,echo=FALSE,result='asis',comment=NA,warning=FALSE}
+library(dplyr)
```

Comment:
you might not need this....you need either this one or the `importFrom` line but not both. Try deleting this and seeing if it still works.